templates/resources/waf_bot_rule_set.md.tmpl

```diff
@@ -0,0 +1,39 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "edgecast_waf_bot_rule_set Resource - terraform-provider-edgecast"
+subcategory: ""
+description: |-
+  
+---
+
+# edgecast_waf_bot_rule_set (Resource)
+Please use the Edgecast API for retrieving specific IDs available for Services, Access Modules, and Delivery Regions.
+A future version of this provider may provide Terraform data sources for these.
+
+## Example Usage
+
+{{tffile "examples/resources/edgecast_waf_bot_rule_set/resource.tf"}}
+
+{{ .SchemaMarkdown | trimspace }}
+
+## Import
+
+To import a resource, create a resource block for it in your configuration:
+
+```terraform
+resource "edgecast_waf_bot_rule_set" "example" {
+  
+}
+```
+
+Now run terraform import to attach an existing instance to the resource configuration:
+
+```shell
+terraform import edgecast_waf_bot_rule_set.example CUSTOMER_ID:ID   
+```
+|             |                                                      |
+|:------------|------------------------------------------------------|
+| `CUSTOMER_ID` | The customer ID of the WAF bot rule set to import. |
```

Comment:
Is 'CUSTOMER_ID' the same as 'ACCOUNT_NUMBER'? We should decide on which terminology to use.